app/controllers/static_pages_controller.rb

```diff
@@ -11,4 +11,9 @@ def program
 
   def contact
   end
+
+  def send_email
+  	ForteMailer.contact_email(params[:subject], params[:first_name], params[:last_name], params[:email], params[:message]).deliver_now
```

Comment:
I would validate these params rather then simply pass them through, but this is fine for now.